config/scout_apm.yml

```diff
@@ -0,0 +1,34 @@
+# This configuration file is used for Scout APM.
+# Environment variables can also be used to configure Scout. See our help docs at http://docs.scoutapm.com#environment-variables for more information.
+common: &defaults
+
+  # key: Your Organization key for Scout APM. Found on the settings screen.
+  # - Default: none
+  key: s4ehrWvArTiWqSpyvLHZ
```

Comment:
is this OK to be public?

Reply:
That is a good question, it should most likely be moved to an ENV, and I assume that you will want to use the key for your account as the main one

Reply:
moved to `SCOUT_KEY='SCOUT_KEY_HERE'`